tests/test_items_base.py

```diff
@@ -61,3 +78,18 @@ def test_item_from_dict_value_error():
     """
     with pytest.raises(ValueError):
         BigItemIncorrect.from_dict({"sub_item": {"name": "hello"}})
+
+
+def test_metadata_default_probability():
+    metadata = Metadata()
+    assert metadata.probability == 1.0
+
+
+def test_metadata_optional_fields():
+    Metadata(
+        dateDownloaded="2022-12-31T13:01:54Z",
+    )
+    Metadata(
+        probability=1.0,
+    )
+    Metadata()
```

Comment:
minor: this is the same test case with line 84

Reply:
Nice catch. In fact, the whole test function can probably be removed, since `Metadata()` working implies that the other two fields can be missing.